java/code/src/org/cobbler/CobblerObject.java

```diff
@@ -461,6 +460,11 @@ public void setKernelOptionsPost(Map<String, Object> kernelOptionsPostIn) {
         modify(SET_KERNEL_OPTIONS_POST, kernelOptionsPostIn);
     }
 
+    /**
+     * @deprecated  This method is no more in cobbler api and is not needed
+     * deprecated version was doing nothing but returning true
+     */
+    @Deprecated
```

Comment:
As I can see this method is protected and only called here in `save()` which you removed above.
I checked the derived classes, nobody call or overwrite `update()`.

So i think it is save to really remove it
What do you think?

Reply:
Agreed. We can remove this method.